src/test/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsTest.java

```diff
@@ -0,0 +1,560 @@
+ */
+public class ManageJournalAbbreviationsTest {
+
+    private ManageJournalAbbreviationsViewModel viewModel;
+    private final String NEWLINE = OS.NEWLINE;
+
+    @Rule
+    public TemporaryFolder tempFolder = new TemporaryFolder();
+
+    private String emptyTestFile;
+    private String testFile1Entries;
+    private String testFile3Entries;
+    private String testFile4Entries;
+    private String testFile5EntriesWithDuplicate;
+
+    private final static JabRefPreferences backupPreferences = Globals.prefs;
+
+
+    @BeforeClass
+    public static void initialize() {
+        Globals.prefs = JabRefPreferences.getInstance();
+    }
+
+    @Before
+    public void setUpViewModel() {
+        // use a new view model for each test so the test methods won't interfere with one another
+        Globals.prefs.putStringList(JabRefPreferences.EXTERNAL_JOURNAL_LISTS, null);
+        viewModel = new ManageJournalAbbreviationsViewModel();
+        emptyTestFile = createTemporaryTestFile("emptyTestFile.txt", "");
+        testFile1Entries = createTemporaryTestFile("testFile1Entries.txt", "Test Entry = TE" + NEWLINE + "");
+        testFile3Entries = createTemporaryTestFile("testFile3Entries.txt",
+                "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE + "");
+        testFile4Entries = createTemporaryTestFile("testFile4Entries.txt",
+                "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE
+                        + "Entry = E" + NEWLINE + "");
+        testFile5EntriesWithDuplicate = createTemporaryTestFile("testFile5Entries.txt",
+                "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "Test Entry = TE" + NEWLINE
+                        + "MoreEntries = ME" + NEWLINE + "EntryEntry = EE" + NEWLINE
+                        + "");
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        Globals.prefs = backupPreferences;
+    }
+
+    @Test
+    public void testInitial() {
+        Assert.assertEquals(viewModel.journalFilesProperty().size(), 0);
+        Assert.assertEquals(viewModel.abbreviationsProperty().size(), 0);
+    }
+
+    @Test
+    public void testInitialWithExternalFiles() throws DuplicatedJournalFileException {
+        // saves 4 test files to the list of external files
+        saveFilesToPreferences();
+
+        viewModel = new ManageJournalAbbreviationsViewModel();
+        viewModel.createFileObjects();
+
+        // check whether all external files from the preferences will
+        // be read when creating the view model
+        Assert.assertEquals(4, viewModel.journalFilesProperty().size());
+        viewModel.changeActiveFile(viewModel.journalFilesProperty().get(3));
+        // remove duplicated entry
+        Assert.assertEquals(5, viewModel.abbreviationsProperty().size());
+    }
+
+    @Test
+    public void testAddFile() throws DuplicatedJournalFileException {
```

Comment:
please rename the tests so that they reflect what you are testing. here `addFileIncreasesCounterOfOpenFiles`